Week1/github.js

```diff
@@ -0,0 +1,94 @@
+/* global createAndAppend, fetchJSON */
+'use strict';
+{
+    const URL_Rep = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    function renderRepository(container, repository) {
+        const dateOfRepository = new Date(repository.updated_at);
+        container.innerHTML = "";
+        const innerTable = createAndAppend("table", container);
+        const tBody = createAndAppend("tbody", innerTable);
+        const tr = createAndAppend('tr', tBody);
+        createAndAppend('td', tr, {text: "Repository:", class: "nameRow"});
+        const tdForLink = createAndAppend('td', tr,);
+        createAndAppend('a', tdForLink, {text: repository.name, href: repository.html_url});
+        const tr1 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr1, {text: "Description:", class: "nameRow"});
+        createAndAppend('td', tr1, {text: repository.description});
+        const tr2 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr2, {text: "Forks:", class: "nameRow"});
+        createAndAppend('td', tr2, {text: repository.forks});
+        const tr3 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr3, {text: "Updated:", class: "nameRow"});
+        createAndAppend('td', tr3, {text: dateOfRepository.toLocaleString()});
+
+
+    }
+    //contributors box
+    async function renderContributors(containerForContributors, repository) {
+        const contributorsURL = repository['contributors_url'];
+        try {
+            const contributors = await fetchJSON(contributorsURL);
+                containerForContributors.innerHTML = "";
+                createAndAppend('p', containerForContributors, { text: 'Contributions' });
+                const contributorsTable = createAndAppend('table', containerForContributors, { id: 'contributorsTable' });
+
+                const contributorsTbody = createAndAppend('tbody', contributorsTable, { id: 'contributorsTbody' });
+               
+                contributors.forEach(contributor => {
+
+                    const contLink = contributor.html_url;
+                    const contImg = contributor.avatar_url;
+                    const contName = contributor.login;
+                    const contNum = contributor.contributions;
+                    const contributorsLink = createAndAppend('a', contributorsTbody, { href: contLink, target: '_blank', class: 'contributorsLink' });
```

Comment:
1. This generates invalid HTML: an `a` element can not be a child of a `tr` element.
2. You are not using the `contributorsLink` class anywhere.